pkg/routeagent_driver/handlers/ovn/handler.go

```diff
@@ -0,0 +1,155 @@
+package ovn
+
+import (
+	"net"
+	"sync"
+
+	"github.com/pkg/errors"
+	"k8s.io/klog"
+
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	cableCleanup "github.com/submariner-io/submariner/pkg/cable/cleanup"
+	"github.com/submariner-io/submariner/pkg/cable/wireguard"
+	clientset "github.com/submariner-io/submariner/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner/pkg/event"
+	"github.com/submariner-io/submariner/pkg/routeagent/cleanup"
+	"github.com/submariner-io/submariner/pkg/routeagent_driver/environment"
+	"github.com/submariner-io/submariner/pkg/util"
+)
+
+type Handler struct {
+	event.HandlerBase
+	mutex                 sync.Mutex
+	config                *environment.Specification
+	smClient              clientset.Interface
+	cableRoutingInterface *net.Interface
+	cleanupHandlers       []cleanup.Handler
+	localEndpoint         *submV1.Endpoint
+	remoteEndpoints       map[string]*submV1.Endpoint
+	isGateway             bool
+}
+
+func NewHandler(env environment.Specification, smClientSet clientset.Interface) *Handler {
+	return &Handler{
+		config:   &env,
+		smClient: smClientSet,
+	}
+}
+
+func (ovn *Handler) GetName() string {
+	return "ovn-hostroutes-handler"
+}
+
+func (ovn *Handler) GetNetworkPlugins() []string {
+	return []string{"OVNKubernetes"}
+}
+
+func (ovn *Handler) Init() error {
+	var err error
+
+	ovn.cableRoutingInterface, err = util.GetDefaultGatewayInterface()
+	if err != nil {
+		klog.Fatalf("Unable to find the default interface on host: %s", err.Error())
+	}
+
+	// For now we get all the cleanups
+	ovn.cleanupHandlers = cableCleanup.GetCleanupHandlers()
```

Comment:
Looks like we are not calling the cleanupHandlers anywhere?

Reply:
oh right, thanks! that part is missing.